pages/03-core/marker-permutations.page.tsx

```diff
@@ -57,10 +57,35 @@ const permutationsForColors = [
   ]),
 );
 
+const permutationsWithStatus = [
+  colorChartsPaletteCategorical1,
+  colorChartsPaletteCategorical2,
+  colorChartsPaletteCategorical3,
+].map((color) =>
+  createPermutations<ChartSeriesMarkerProps>([
+    {
+      type: [
```

Comment:
nit: I would extract marker types to a separate var to be reused between permutation definitions